src/redux/slices/userSlice.ts

```diff
@@ -21,10 +25,14 @@ export const userSlice = createSlice({
       ...state,
       ...action.payload,
     }),
-    userLogout: (_state) => ({ walletType: wallet.WalletType.NONE }),
+    userLogout: (_state) => ({
+      walletType: wallet.WalletType.NONE,
+      displayedProfileId: undefined,
```

Comment:
If this were right, this line would be redundant, but I don't think this is right. It should be possible to click on a profile from the the `Discover` feed even when you're not logged in, so it doesn't really make sense for logging out to affect the `displayedProfileId` state. If this is the correct behavior, this line should be `displayedProfileId: state.displayedProfileId`.